visidata/fill.py

```diff
@@ -9,7 +9,7 @@ def fillNullValues(vd, col, rows):
     oldvals = [] # for undo
     isNull = col.sheet.isNullFunc()
     n = 0
-    rowsToFill = [id(r) for r in rows]
+    rowsToFill = { id(r):True for r in rows }
```

Comment:
I generally use `{id(r):r for r in rows}` but a set would be even better: `set(id(r) for r in rows)`